### PR TITLE
Changes to activate all raydium pools on production

### PR DESCRIPTION
**Pull request by godmodegalactus**

### CODE REVIEW

**Comment by godmodegalactus** [comment]
> Does that work without #10 ? Router can keep up and update prices fast enough ?

Yes it does work without #10

File: bin/autobahn-router/src/prometheus_sync.rs [hunk 1]
```diff
@@ -29,29 +29,29 @@ impl PrometheusSync {
...
-            loop {
-                let Ok((mut stream, _addr)) = listener.accept().await else {
-                    error!("Error accepting prometheus stream");
-                    tokio::time::sleep(Duration::from_millis(1)).await;
-                    continue;
-                };
+            let mut router: Router<()> = Router::new();
+            router = router.route("/metrics", routing::get(Self::get_prometheus_stream));
+
+            let cors = CorsLayer::new()
```
[the remark this answers is not in the document]
Comment (godmodegalactus) [reply bin/autobahn-router/src/prometheus_sync.rs:42]: Yes i will remove cors

File: bin/autobahn-router/template-config.toml [hunk 2]
```diff
@@ -77,26 +79,61 @@ dedup_queue_size = 50000
...
+request_timeout_in_seconds = 300
+
+[[sources.grpc_sources]]
+name = "router-DFW"
+connection_string = "$DFW_RPC_HTTP_URL_WITHOUT_TOKEN"
+token = "$AMS_RPC_TOKEN"
+retry_connection_sleep_secs = 30
+
+[[sources.quic_sources]]
+name = "quic-client"
```
Comment (riordanp) [inline bin/autobahn-router/template-config.toml:111]: minor: quic source should have region suffix as above with grpc

Comment (godmodegalactus) [reply bin/autobahn-router/template-config.toml]: fixed

**Pull request merged**